packages/terra-action-header/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Action Header
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-action-header.svg)](https://www.npmjs.org/package/terra-action-header)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The terra-action-header component is a header bar that contains sockets for placing actionable items such as buttons and hyperlinks.
```

Comment:
> contains sockets 

This should probably updated. The props do not indicate there are sockets in this component.

Reply:
Rephrased [here](https://github.com/cerner/terra-core/commit/b1e09f51f44a1123922961a673cc07c9e2925f33).